QUICKSTART.md

```diff
@@ -0,0 +1,251 @@
+      "repository": "hypha-space/lenet",
+      "type": "vision-classification",
+      "filenames": [
+        "config.json",
+        "model.safetensors",
+        "configuration_lenet.py",
+        "modeling_lenet.py"
+      ],
+      "token": "TBD"
+    },
+    "preprocessor": {
+      "repository": "hypha-space/lenet",
+      "filenames": ["preprocessor_config.json"],
+      "token": "TBD"
+    },
+    "dataset": {
+      "dataset": "mnist"
+    },
+    "inner_optimizer": { "learning-rate": 1e-3 },
+    "outer_optimizer": { "learning-rate": 0.7, "momentum": 0.9 },
+    "resources": {
+      "num_workers": 2,
+      "worker": [
+        { "type": "GPU", "min": 10.0 },
+        { "type": "CPU", "min": 1.0 },
+        { "type": "Memory", "min": 1.0 }
+      ],
+      "parameter_server": [
+        { "type": "CPU", "min": 1.0 },
+        { "type": "Memory", "min": 1.0 }
+      ]
+    }
+  }
+}'
+```
+
+##### 2.2.3 Set up worker nodes
+
+You can generate worker configs using `hypha-worker init`:
+
+```bash
+./hypha-worker init -n worker1 && \
+./hypha-worker init -n worker2 && \
+./hypha-worker init -n worker3
+```
+
+##### 2.2.4 Set up data node
+
+You can generate a data config using `hypha-data init`:
+
+```bash
+./hypha-data init -n data1 --dataset-path="mnist"
+```
+
+We will download the dataset below.
+
+---
+
+Done. Now all your Nodes are set up correctly.
+
+---
+
+#### 2.3 setting up the accelerate driver
+
+> **NOTE**: We're currently relying on a hard coded path for this driver and dont have a proper bundling process in place.
+> It is important to keep the path as is (`./drivers/hypha-accelerate-driver` relative from your `hypha-worker` binary).
+
+Copy over the driver folder from your hypha repository you cloned earlier.
+
+```bash
+cp -r ../path/to/repo/drivers/hypha-accelerate-driver ./drivers`
+```
+
+#### 2.4 Download the training dataset
+
+Section TBD
+
+```bash
+mkdir imagnet
+curl ...
+```
+
+---
+
+## Running hypha for training
+
+Now that everything is set up, you can start the hypha nodes and begin your training.
+
+In different terminals, first start the gateway:
+
+1. `./hypha-gateway run -c gateway-config.toml`
+
+Then start the worker and data nodes (gateway needs to be up first):
+
+1. `./hypha-worker run -c worker1-config.toml`
+1. `./hypha-worker run -c worker2-config.toml`
+1. `./hypha-worker run -c worker3-config.toml`
+1. `./hypha-data run -c data-config.toml`
+
+Lastly, once all nodes above are running fine, start the scheduler to start a training:
+
+1. `./hypha-scheduler run -c scheduler-config.toml`
+
+### Expected output:
+
+Section TBD
+
+---
+
+## Troubleshooting
+
+| Issue               | Possible Fix                             |
+| ------------------- | ---------------------------------------- |
+| App won’t start     | Check config file formatting             |
+| Port already in use | Change `port:` in `config.yaml`          |
+| Permission errors   | Try running with `sudo` or fix ownership |
```

Comment:
Did we encounter permission errors during testing? Right now we only read files in our implementation, I wouldn't advise to run with `sudo`.